app/views/recipes/show.html.erb

```diff
@@ -1,10 +1,51 @@
-<p style="color: green"><%= notice %></p>
+<%= stylesheet_link_tag "recipe", "data-turbo-track": "reload" %>
+<% classes_row = ["colored", "natural"] %>
+<% i = 0 %>
+<img class ="icone-recipe" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQOWry5OIVtLKR-JVVI4cjFYMPJm5sSFNOF4w&usqp=CAU" />
 
-<%= render @recipe %>
-
-<div>
-  <%= link_to "Edit this recipe", edit_recipe_path(@recipe) %> |
-  <%= link_to "Back to recipes", recipes_path %>
-
-  <%= button_to "Destroy this recipe", @recipe, method: :delete %>
+<h1 class="recipe-title"> <%= @recipe.name %> </h1>
+<div class="recip-general-info">
+  <div class="recip-process">
+    <p>Preparation time: <%= @recipe.preparation_time %> </p>
+    <p>Cooking time: <%= @recipe.cooking_time %> </p>
+    <p> Steps go here... </p>
+  </div>
+  <div class="public-state">
+    <label for="public"> Public</label>
+    <input type="range" name="public"  min=0 max=1 value=<%= @recipe.public %> disabled>
+  </div>
```

Comment:
I can't change the public state here.

Reply:
The view display only information of the recipe , don't allow user to change anything.